controller/script/simulate-proxy/main.go

```diff
@@ -98,6 +99,26 @@ var (
 		FramesSent: 4,
 	}
 	ports = []uint32{3333, 6262}
+
+	// these values should be one order of magnitude greater than
+	// the controller's Prometheus buckets, because the proxy will
+	// report them in tenths of a millisecond.
+	latencyBucketBounds = [26]uint32{
```

Comment:
What unit are these values actually in?  I'm assuming these are milliseconds based on the above comment, but it could be clearer.  Also, I tend to prefer that comments are Capitalized Properly so that I have a clearer indication of if , for instance, the first sentence of this comment was just lost or not (as it is, i'm slightly suspicious that a this happened ;)

Perhaps:
> latencyBucketBounds holds the maximum value (inclusive, in tenths of a millisecond) that may be counted in a given histogram bucket.
>
> These values are one order of magnitude greater than the controller's Prometheus buckets, because the proxy will reports latencies in tenths of a millisecond.